Gemfile

```diff
@@ -8,8 +8,10 @@ gem 'rails'
 #gem 'sqlite3-ruby', :require => 'sqlite3'
 
 # Use unicorn as the web server
+gem 'byebug'
```

Comment:
This gem should be used only in development mode. Please add it to the `development` section